src/fragments/lib/analytics/flutter/getting-started/10_preReq.mdx

```diff
@@ -1,5 +1,37 @@
+import flutter0 from "/src/fragments/lib/analytics/flutter/sqlite3_wasm.mdx";
+
+<BlockSwitcher>
+
+<Block name="Stable (Mobile)">
+
 * [Install and configure Amplify CLI](https://docs.amplify.aws/cli/start/install)
 * A Flutter application targeting Flutter SDK >= 2.10.0 (stable version) with Amplify libraries integrated
   * An iOS configuration targeting at least iOS 11.0
   * An Android configuration targeting at least Android API level 21 (Android 5.0) or above
   * For a full example please follow the [project setup walkthrough](/lib/project-setup/create-application)
+<br/><br/>
```

Comment:
can we remove this spaces please?

Reply:
I think these spaces are really important, as otherwise when switching between either mode the entire screen shifts making it easy to lose your place in the document.

Reply:
got it - makes sense, Kyle! Resolving this convo